framework/src/modules/interoperability/mainchain/commands/sidechain_registration.ts

```diff
@@ -216,10 +216,7 @@ export class SidechainRegistrationCommand extends BaseInteroperabilityCommand {
 		await chainSubstore.set(context, chainID, sidechainAccount);
 
 		// Add an entry in the channel substore
-		const messageFeeTokenID = {
-			chainID: utils.intToBuffer(1, 4),
-			localID: utils.intToBuffer(0, 4),
-		};
+		const messageFeeTokenID = Buffer.from('0000000000000011', 'hex');
```

Comment:
where did you get this value? shouldn't this be `0000000100000000`

Reply:
btw mainchain ID / mainchain token ID should be computed like
```
const networkID = this._config.ownChainID.slice(0, 1);
// 3 bytes for remaining chainID bytes
const mainchainID = Buffer.concat([networkID, Buffer.alloc(CHAIN_ID_LENGTH - 1, 0)]);
const mainchainTokenID = Buffer.concat([mainchainID, Buffer.alloc(LOCAL_ID_LENGTH, 0)];
```

Reply:
we can have a util func for this

Reply:
https://github.com/LiskHQ/lisk-sdk/issues/7701